Tests/SwiftProtobufTests/Test_AllTypes_Proto3.swift

```diff
@@ -777,13 +777,17 @@ class Test_AllTypes_Proto3: XCTestCase, PBTestHelpers {
         assertDecodeFails([253, 1, 77])
         assertDecodeFails([254, 1, 78])
         assertDecodeFails([255, 1, 79])
-        assertDebugDescription("Proto3TestAllTypes(repeatedInt32:[1])") {(o: inout MessageTestType) in
+        // WHY ISN'T THIS LIKE PROTO2?
+        //   "Proto3TestAllTypes:\nrepeated_int32: 1\n"
+        assertDebugDescription("Proto3TestAllTypes:\nrepeated_int32: [1]\n") {(o: inout MessageTestType) in
```

Comment:
Ok, sounds like this is all follow up for the TextFormat changes in general, and not specific to this CL, so I'll go ahead and land this so I can finish cleaning out the info the plugin is capturing that was just to support this.